lib/desktopFile.js

```diff
@@ -125,8 +123,8 @@ const Editor = new Lang.Class({
         this._path = userFullPath.get_path();
         this._metadataPath = this._path + '.json';
 
-        let [_, fullPath] = this._keyFile.load_from_data_dirs(path,
-                GLib.KeyFileFlags.KEEP_TRANSLATIONS);
+        let fullPath = this._keyFile.load_from_data_dirs(path,
+                GLib.KeyFileFlags.KEEP_TRANSLATIONS)[1];
```

Comment:
Unfortunately not :( There actually doesn't seem to be any default rules about vertical alignment of parameters, so ESLint kind of just lets them go anywhere. We could write a plugin though.